### DISCUSSION TITLE
Can you add the panorama mode back it was perfect

### DISCUSSION THREAD

**Discussion by Frozen-Fang** [FossifyOrg/General-Discussion#409] [General]
I use the panorama mode a decent amount on the original version of this app, and seeing it removed for no reason on this version sucks, this would be without a doubt the best app if it still had it, every other app I have tried does panorama terrible except this app before it was removed

**Comment by Aga-C** [comment]
It's already reported here: https://github.com/FossifyOrg/Gallery/issues/42.
